.travis.yml

```diff
@@ -42,6 +42,13 @@ script:
   - npm run lint
   - "if [ $COVERAGE ]; then npm run cover && bash <(curl -s https://codecov.io/bash) -f coverage/lcov.info; else npm run test; fi"
 
+after_success:
+  - npm run semantic-release
```

Comment:
I think they check this. This was automatically setup by their script. Otherwise this PR would have been released :D